trex.org

```diff
@@ -252,138 +252,82 @@
-    radial function $R_s$. The radial function is a linear combination of
-    $N_{\text{prim}}$ /primitive/ functions that can be of type
-    Slater ($p=1$) or Gaussian ($p=2$),
-    parameterized by exponents $\gamma_{ks}$ and coefficients $a_{ks}$:
-    \[
-    R_s(\mathbf{r}) = \mathcal{N}_s \vert\mathbf{r}-\mathbf{R}_A\vert^{n_s}
-    \sum_{k=1}^{N_{\text{prim}}} a_{ks}\, f_{ks}(\gamma_{ks},p)\,
-    \exp \left( - \gamma_{ks}
-    \vert \mathbf{r}-\mathbf{R}_A \vert ^p \right).
-    \]
-
-    In the case of Gaussian functions, $n_s$ is always zero.
-
-    Different codes normalize functions at different levels. Computing
-    normalization factors requires the ability to compute overlap
-    integrals, so the normalization factors should be written in the
-    file to ensure that the file is self-contained and does not need the
-    client program to have the ability to compute such integrals.
-
-    Some codes assume that the contraction coefficients are for a linear
-    combination of /normalized/ primitives. This implies that a normalization
-    constant for the primitive $ks$ needs to be computed and stored. If
-    this normalization factor is not required, $f_{ks}=1$.
-
-    Some codes assume that the basis functions are normalized. This
-    implies the computation of an extra normalization factor, $\mathcal{N}_s$.
-    If the the basis function is not considered normalized, $\mathcal{N}_s=1$.
-
-    All the basis set parameters are stored in one-dimensional arrays.
+  Different computational chemistry codes apply normalization at
+  various levels. The normalization of basis functions requires the
+  evaluation of diagonal overlap integrals, which might not always be
+  supported by the client software. For this reason, normalization
+  constants are explicitly provided in the TREXIO files to
+  ensure portability and completeness.
+
+  We consider here basis functions centered on nuclei. It is
+  therefore possible to introduce *dummy atoms* (artificial centers
+  with no physical meaning) solely for the purpose of placing basis
+  functions at arbitrary locations in space.
+
+  The atomic basis set is defined as a list of shells. Each shell
+  $s$ is centered on a point $\mathbf{R}_A$, associated with a given
+  angular momentum $\ell_s$, and characterized by a radial function
+  $R_s(\mathbf{r})$. Note that since the angular functions are
+  defined as regular solid harmonics that include the $r^{\ell_s}$ factor,
+  the radial functions in TREXIO differ from the textbook definition
+  by a factor of $r^{-\ell_s}$.
+
+  The radial function is a linear combination of
+  $N_{\text{prim}}$ *primitive* functions. These primitives may be
+  of Slater-type or Gaussian-type, and are parameterized by
+  exponents $\gamma_{ks}$ and contraction coefficients $a_{ks}$.
+
+  Slater orbitals are written as:
+
+  \[
+  R_s^{\text{STO}}(\mathbf{r}) = \mathcal{N}_s  \left| \mathbf{r} - \mathbf{R}_A \right|^{n_s-1-\ell_s}
```

Comment:
It's better to use the same convention as for Gaussians
```suggestion
  R_s^{\text{STO}}(\mathbf{r}) = \mathcal{N}_s  \left| \mathbf{r} - \mathbf{R}_A \right|^{n_s}
```

Reply:
OK, so then I will go back to a single expression for GTOs and STOs.